deploy.py

```diff
@@ -154,11 +175,13 @@
             kernel_address=0x1000,
             nordic_dfu=True,
         ),
+    "cw310_opensk":
+        cw310_opensk_board,
 }
 
 # The following value must match the one used in the file
 # `src/entry_point.rs`
-APP_HEAP_SIZE = 90000
+APP_HEAP_SIZE = 90_000
```

Comment:
I see. I wondered how you came up with this specific heap limit, do you have some measurements with the certified versions that give a rough estimate how much heap is needed for allocation? Because this seems to be a rather large value.